### PR TITLE
Migrate flutter_contacts to android v2 embedding.

### PR DESCRIPTION
**Pull request by lidongze91**
Migrate the plugin to android v2 embedding and migrate androidx for example app.

### CODE REVIEW

**Comment by lidongze91** [comment]
@lukasgit Hi Lukas, can you do an review? Thanks.

File: android/src/main/java/flutter/plugins/contactsservice/contactsservice/ContactsServicePlugin.java [hunk 1]
```diff
@@ -22,6 +23,9 @@
 import android.provider.ContactsContract.RawContacts;
 import android.text.TextUtils;
 import android.util.Log;
+import androidx.annotation.NonNull;
```
Comment (mehmetf) [inline android/src/main/java/flutter/plugins/contactsservice/contactsservice/ContactsServicePlugin.java:26]: Do you have to use this annotation?

Comment (lidongze91) [reply android/src/main/java/flutter/plugins/contactsservice/contactsservice/ContactsServicePlugin.java]: Deleted.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## [0.4.0] - March 30, 2020
+
+* Migrate the plugin to android v2 embedding.
+* Migrate AndroidX for the example app.
```
Comment (mehmetf) [inline CHANGELOG.md:4]: Thanks for doing this but let's keep the scope of the PR small. This is just about migrating to v2. Let's remove all androidx references.

**Comment by mehmetf** [comment]
@zhenqiu1101 please take a look as well.

File: CHANGELOG.md (hunk 2 above)
Comment (lidongze91) [reply CHANGELOG.md]: @mehmetf Deleted.

**Pull request merged**